### PR TITLE
WIP: Cross-compile 213

### PR DESCRIPTION
**Pull request by philvarner**
# Overview

Cross-compile with 2.13 for non-Spark packages

## Checklist

- [ ] [./CHANGELOG.md](https://github.com/locationtech/geotrellis/blob/master/CHANGELOG.md) updated, if necessary. Link to the issue if closed, otherwise the PR.
- [ ] [Module Hierarchy](https://github.com/locationtech/geotrellis/blob/master/docs/guide/module-hierarchy.rst) updated, if necessary
- [ ] `docs` guides update, if necessary
- [ ] New user API has useful Scaladoc strings
- [ ] Unit tests added for bug-fix or new feature

## Demo

Optional. Screenshots/REPL

## Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

Closes #XXX

### CODE REVIEW

File: proj4/src/main/scala/geotrellis/proj4/io/wkt/WKT.scala [hunk 1]
```diff
@@ -22,7 +22,7 @@ import scala.io.Source
 
 object WKT {
   private val wktResourcePath = "/proj4/wkt/epsg.properties"
-  lazy val parsed: Map[Int, WktCS] = records.mapValues(WKTParser.apply)
+  lazy val parsed: Map[Int, WktCS] = records.mapValues(WKTParser.apply).toMap
```
Comment (yurique) [inline proj4/src/main/scala/geotrellis/proj4/io/wkt/WKT.scala:25]: `mapValues` is deprecated in 2.13 

> Use .view.mapValues(f). A future version will include a strict version of this method (for now, .view.mapValues(f).toMap).

**Comment by yurique** [comment]
another thing is `Traversable` and `TraversableOnce` – these are deprecated in 2.13 as well. 

```scala
  @scala.deprecated("Use Iterable instead of Traversable", "2.13.0")
  type Traversable[+A] = scala.collection.Iterable[A]
```

```scala
  @scala.deprecated("Use IterableOnce instead of TraversableOnce", "2.13.0")
  type TraversableOnce[+A] = scala.collection.IterableOnce[A]
```

File: proj4/src/main/scala/geotrellis/proj4/io/wkt/WKT.scala (hunk 1 above)
Comment (philvarner) [reply proj4/src/main/scala/geotrellis/proj4/io/wkt/WKT.scala:25]: Yes -- this is just to maintain backwards compatibility with 2.11 and 2.12 with the fewest changes possible.  There are hundreds if not thousands of deprecation warnings when compiling with 2.13 😬

**Comment by pomadchin** [comment]
Thanks a lot! Closing it in favor of https://github.com/locationtech/geotrellis/pull/3294

**Pull request closed without merging**